### ISSUE TITLE
Correct the tree by org statistic

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-reporting#62]
We should do the same thing for tree as did for capture:
```
 "captures": {
    "total": 1517305,
    "captures": [
      {
        "name": "FCCFED",
        "number": "411044"
      },
      {
        "name": "FCCYAR",
        "number": "337863"
      },
      {
        "name": "FCCEIC",
        "number": "152478"
      }
    ]
```

**Comment by dadiorchen** [comment]
The way to filter by org, please refer to this: https://github.com/Greenstand/treetracker-api/pull/128

**Comment by kushanshamika** [comment]
I will work on this issue

**Comment by Kpoke** [comment]
Hi @kushanshamika , can you hold off on this issue?

**Comment by Kpoke** [comment]
@dadiorchen , this is the current organization filter we have, I think this might be an issue for the airflow dag, to update which organization_id to pick up. What do you think?
![image](https://user-images.githubusercontent.com/28721558/212499784-579c89e9-8d02-4d1f-99da-496b23334e06.png)

**Comment by dadiorchen** [comment]
Yes, this is how I filter by org on the matching tool, I'm not sure if it is good for the airflow dag, in general, I think it should work.

**Comment by Kpoke** [comment]
I am thinking that the planting_organization_uuid field that would be populated in the reporting schema would be that of the grower account (entity), we'd make that update in the airflow dag

**Comment by dadiorchen** [comment]
Do you mean so we don't need those complicated join to trace up the relationship, instead, we can use the planting_organization_uuid directly?

**Comment by Kpoke** [comment]
Yeah, the complicated join would be done on airflow, when populating the schema

**Comment by dadiorchen** [comment]
Thanks @Kpoke , that's the right way to go.

**Comment by dadiorchen** [comment]
BTW, which job on airflow would do the thing you mentioned above?

**Comment by Kpoke** [comment]
The tree reporting and the capture reporting dags
